### PR TITLE
replacing folder node name default character with hyphen

### PR DESCRIPTION
**Pull request by kmurugulla**
@badvision 
Modified code to create folder node names with "-" as separator instead of "_". Please review.

### CODE REVIEW

File: bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/AssetIngestor.java [hunk 1]
```diff
@@ -438,7 +438,7 @@ default String getNodeName() {
                 String extension = StringUtils.substringAfterLast(name, ".");
                 return JcrUtil.createValidName(baseName) + "." + JcrUtil.createValidName(extension);
```
Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/AssetIngestor.java]: why wouldn't this instance of createValidName use the same convention?

**Comment by davidjgonzalez** [comment]
I think this makes sense - as hyphens are the common replacement for spaces in AEM node names.
That being said, I'm not sure if this would be problematic/annoying for customers that have ingested a ton of assets using the old format?

An option to select the valid naming scheme seems overkill to support the old behavior, but... idk. WDYT? @badvision ?

**Comment by davidjgonzalez** [comment]
Is it right hat only one instance of getValidName uses this rule? (See my first comment)

**Comment by badvision** [comment]
Oh I didn't catch that @davidjgonzalez  -- @kmurugulla can you update the conversion for all parts of the path so it's more consistent?

**Pull request merged**